ci/cron/daily-compat.yml

```diff
@@ -116,3 +116,61 @@ jobs:
       - template: ../daily_tell_slack.yml
         parameters:
           success-message: '$(cat $(Build.StagingDirectory)/perf-results.json | jq . | jq -sR ''"perf for ''"$COMMIT_LINK"'':```\(.)```"'')'
+
+  - job: perf_http_json
+    timeoutInMinutes: 120
+    pool:
+      name: "linux-pool"
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          source $(bash_lib)
+
+          SCENARIOS="\
+           com.daml.http.perf.scenario.CreateCommand \
+           com.daml.http.perf.scenario.ExerciseCommand \
+           com.daml.http.perf.scenario.CreateAndExerciseCommand \
+           com.daml.http.perf.scenario.AsyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryNewAcs \
+           com.daml.http.perf.scenario.SyncQueryVariableAcs \
+          "
+
+          bazel build //docs:quickstart-model
+          DAR="${PWD}/bazel-bin/docs/quickstart-model.dar"
+
+          JWT="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU"
+
+          START=$(git log -n1 --format=%cd --date=format:%Y%m%d).$(git rev-list --count HEAD).$(Build.BuildId).$(git log -n1 --format=%h --abbrev=8)
```

Comment:
@garyverhaegen-da what exactly does the buildid buy us? iirc Azure GCs old builds pretty quickly so whatever information in there seems to be lost fairly quickly.

Reply:
It serves the same purpose as the `0` we have in version numbers for that segment: uniqueness. It seems very plausible we'll run the daily tests on the same commit multiple times (say, over the weekend).

Reply:
Thanks for the explanation!

Reply:
However I would just keep the last perf test results archive in this case.

Reply:
We're talking ~100kb per run, I think we can afford a few extra ones. More data seems better in general, and multiple runs of the same code gives us some way to assess reproducibility.